src/app/spans.rs

```diff
@@ -66,11 +85,14 @@ impl SpanConverter {
             kind: self.kind,
             start_time: Some(span.start.into()),
             end_time: Some(span.end.into()),
-            attributes: None, // TODO: attributes
+            attributes: Some(oc::span::Attributes {
+                attribute_map: attributes,
+                dropped_attributes_count: 0,
```

Comment:
>  Attributes can be discarded because their keys are too long or because there are too many attributes

Are these things that need to be checked when making `attributes` above?

Reply:
The list of attributes that we populate is hardcoded so I don't think we need to worry about this aspect of the API.